gcloud-java-dns/src/test/java/com/google/gcloud/dns/RecordSetTest.java

```diff
@@ -26,20 +26,20 @@
 
 import java.util.concurrent.TimeUnit;
 
-public class DnsRecordTest {
+public class RecordSetTest {
 
   private static final String NAME = "example.com.";
   private static final Integer TTL = 3600;
   private static final TimeUnit UNIT = TimeUnit.HOURS;
   private static final Integer UNIT_TTL = 1;
-  private static final DnsRecord.Type TYPE = DnsRecord.Type.AAAA;
-  private static final DnsRecord record = builder(NAME, TYPE)
+  private static final RecordSet.Type TYPE = RecordSet.Type.AAAA;
+  private static final RecordSet recordSet = builder(NAME, TYPE)
       .ttl(UNIT_TTL, UNIT)
       .build();
 
   @Test
   public void testDefaultDnsRecord() {
-    DnsRecord record = builder(NAME, TYPE).build();
+    RecordSet record = builder(NAME, TYPE).build();
```

Comment:
Nit (only if you have time): here and in the following of this class variable names should be like `recordSet`, not `record`.